verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -110,7 +120,21 @@ export default class CasesController {
         const location = req.body['location'];
         if (!location?.geometry?.latitude || !location.geometry?.longitude) {
             for (const geocoder of this.geocoders) {
-                const features = await geocoder.geocode(location?.query);
+                const opts: GeocodeOptions = {};
+                if (location['limitToResolution']) {
+                    opts.limitToResolution =
+                        Resolution[
+                            location[
+                                'limitToResolution'
+                            ] as keyof typeof Resolution
+                        ];
+                    if (!opts.limitToResolution) {
+                        throw new InvalidParamError(
+                            `invalid limitToResolution: ${location['limitToResolution']}`,
+                        );
```

Comment:
DYT this logic should be here or in the geocoder?

Reply:
Both really, the cases controller deal with API types conversion and the mapbox geocoder deal with mapbox types conversion.